doc/user/content/sql/show-sources.md

```diff
@@ -16,12 +16,26 @@ instances.
 Field | Use
 ------|-----
 _schema&lowbar;name_ | The schema to show sources from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
+**MATERIALIZED** | Only return materialized sources, i.e. those with [indexes](../create-index). Without specifying this option, this command returns all sources, including non-materialized sources.
+**FULL** | Return details about your sources.
 
 ## Details
 
-### Output format
+### Output format for `SHOW FULL SOURCES`
 
-`SHOW SOURCES`'s output is a table with one column, `name`.
+`SHOW FULL SOURCES`'s output is a table, with this structure:
+
+```nofmt
+ name  | type | materialized
+-------+------+--------------
+ ...   | ...  | ...
+```
+
+Field | Meaning
+------|--------
+**name** | The name of the source
+**type** | Whether the source was created by the user or the system
```

Comment:
If "user" and "system" are the values of this column, let's code format them; if those aren't the literal values, enclose them in parentheses next to the description in text, e.g. 


> Whether the source was created by the `user` or the `system`
or
> Whether the source was created by the user (`some user value`) or the system (`some system value`)